src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
Consider using a loop in your templating engine or a script to generate calendar days to avoid repetitive code and make maintenance easier. Having many repeated elements like `calendar__day` can be error-prone if you ever need to change them.